example/ck_tile/03_gemm/gemm_reduce.cpp

```diff
@@ -0,0 +1,1009 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2024-2025, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
We can skip the copyright year at all. 
```suggestion
// Copyright (c) Advanced Micro Devices, Inc. All rights reserved.
```